packages/Search/src/details/DTK_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -57,11 +57,19 @@ struct DistributedSearchTreeImpl
         Kokkos::View<double *, DeviceType> *distances_ptr = nullptr );
 
     template <typename Query>
-    static void deviseStrategy( Point epsilon,
-                                Kokkos::View<Query *, DeviceType> queries,
+    static void deviseStrategy( Kokkos::View<Query *, DeviceType> queries,
```

Comment:
Please add a comment on what `deviseStrategy` and `reassessStrategy` do. It hasn't been that long and yet I don't have any clue on what `deviseStrategy` is supposed to do.